config/pre-install/v0.19.0/storage-version-migration.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: storage-version-migration-knative-gcp
```

Comment:
@capri-xiyue Do we want to change this name to avoid conflict with https://github.com/google/knative-gcp/blob/master/config/pre-install/v0.18.0/storage-version-migration.yaml#L18

Reply:
If a user upgraded from 0.17 -> 0.18 and run `pre-install/v0.18.0`, then the job `storage-version-migration-knative-gcp` will be created, then if the user want to upgrade from 0.18 -> 0.19, there will be a conflict since job has less args than the previous one.

Reply:
Let's change it to storage-version-migration-knative-gcp-v0.19.0? Please also change the `labels` and make it match the name.

Reply:
Let's change it to` storage-version-migration-knative-gcp-v0.19.0` first.

Reply:
Done.